### PR TITLE
New package: HamRadioDeluxe.HamRadioDeluxe version v6.8.0.372

### PR DESCRIPTION
**Pull request by xylo04**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111638&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
+Installers:
+- InstallerUrl: https://downloads.hamradiodeluxe.com/setupHRD68_372.exe?ucacid=1599084031.255368
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloads.hamradiodeluxe.com/setupHRD68_372.exe
```

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
+PackageLocale: en-US
+Publisher: HRD Software, LLC.
+PackageName: Ham Radio Deluxe
+License: Proprietary
+Copyright: Copyright � 2005-2022 by HRD Software, LLC.
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml:10]:
```suggestion
Copyright: Copyright © 2005-2022 by HRD Software, LLC.
```

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
+PackageLocale: en-US
+Publisher: HRD Software, LLC.
+PackageName: Ham Radio Deluxe
+License: Proprietary
+Copyright: Copyright � 2005-2022 by HRD Software, LLC.
+ShortDescription: Ham Radio Deluxe
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml:11]:
```suggestion
ShortDescription: A Complete Software Suite for Amateur Radio Operators
```

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.8.0.372
```
This should match control panel exactly. After changing this, the folder name will need to be updated.

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.installer.yaml:5]:
```suggestion
PackageVersion: 6.8.0.372
```
This should match control panel exactly. After changing this, the folder name will need to be updated.

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.yaml:5]:
```suggestion
PackageVersion: 6.8.0.372
```
This should match control panel exactly. After changing this, the folder name will need to be updated.

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
+PackageLocale: en-US
+Publisher: HRD Software, LLC.
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml:7]:
```suggestion
Publisher: HRD Software, LLC.
# PublisherUrl:
PublisherSupportUrl: https://support.hamradiodeluxe.com/
PrivacyUrl: https://www.hamradiodeluxe.com/help/privacy-policy/
# Author:
```

File: manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372
+PackageLocale: en-US
+Publisher: HRD Software, LLC.
+PackageName: Ham Radio Deluxe
+License: Proprietary
+Copyright: Copyright � 2005-2022 by HRD Software, LLC.
+ShortDescription: Ham Radio Deluxe
+ManifestType: defaultLocale
```
Comment (Trenly) [inline manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.locale.en-US.yaml:12]:
```suggestion
PurchaseUrl: https://www.hamradiodeluxe.com/buy/
Documentations:
- DocumentLabel: User Guides
  DocumentUrl: https://support.hamradiodeluxe.com/support/solutions/51000018230
# ReleaseNotes:
ReleaseNotesUrl: https://releasenotes.hamradiodeluxe.com/
ManifestType: defaultLocale
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by xylo04** [comment]
Thanks for the review!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**